test/test_subdir_package.py

```diff
@@ -0,0 +1,54 @@
+import os
+
+import jinja2
+
+from . import utils
+from .template_projects import TemplateProject
+from .template_projects.c import spam_c_template
+
+subdir_package_project = TemplateProject()
+
+subdir_package_project.files['src/spam/spam.c'] = jinja2.Template(spam_c_template)
```

Comment:
`template_projects.c.spam_c_template` could be `SPAM_C_TEMPLATE` to indicate it's some kind of constant?

Reply:
👍